src/components/Domain/service.ts

```diff
@@ -280,6 +281,13 @@ const DomainService = {
     );
     return response.status === 200;
   },
+  callDomain(url: string) {
+    try {
+      axios.get(`https://${url}`);
```

Comment:
@Abhimanyu121 try won't work on async call.
Do something like this
```suggestion
      axios.get(`https://${url}`).then(res => console.log("CALLING DOMAIN TO GENERATE CERTS")).catch(err => console.log);
```